en/theme/material/assets/css/theme.css

```
@@ -900,3 +900,111 @@ html .md-typeset .superfences-tabs > label:hover {
+    background: transparent;
+    border: 0;
+    border-radius: 0.25rem;
+    color: #6a7a8d;
+    cursor: pointer;
+    transition: background 0.15s ease, color 0.15s ease;
+}
+
+.rebrand-notice__dismiss svg {
+    width: 1rem;
+    height: 1rem;
+    fill: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)